diff --git a/dist/components/SidebarSublist.astro b/dist/components/SidebarSublist.astro
--- a/dist/components/SidebarSublist.astro
+++ b/dist/components/SidebarSublist.astro
@@ -25,6 +25,9 @@
 						{...entry.attrs}
 					>
 						<span>{entry.label}</span>
+						{entry.attrs.target === '_blank' && (
+							<Icon name="external" class="external-icon" />
+						)}
 						{entry.badge && (
 							<Badge
 								variant={entry.badge.variant}
@@ -58,3 +61,7 @@
 </ul>
 
 <style>
+	.external-icon {
+		vertical-align: middle;
+	}
+
